client-src/elements/chromedash-gate-column.js

```diff
@@ -6,15 +6,23 @@ import {showToastMessage, findProcessStage} from './utils.js';
 import {SHARED_STYLES} from '../sass/shared-css.js';
 
 
-export const STATE_NAMES = [
-  [7, 'No response'],
-  [1, 'N/a or Ack'],
-  [2, 'Review requested'],
-  [3, 'Review started'],
-  [4, 'Needs work'],
-  [8, 'Internal review'],
-  [5, 'Approved'],
-  [6, 'Denied'],
+export const PREPARING = 0;
+export const REVIEW_REQUESTED = 2;
+export const STATE_NAMES = {
+  NO_RESPONSE: [7, 'No response'],
+  NA: [1, 'N/a or Ack'],
+  REVIEW_STARTED: [3, 'Review started'],
+  NEEDS_WORK: [4, 'Needs work'],
+  INTERNAL_REVIEW: [8, 'Internal review'],
+  APPROVED: [5, 'Approved'],
+  DENIED: [6, 'Denied'],
+};
```

Comment:
A possible suggestion here: it might be more readable when referencing objects like this to have a nested object with two properties, like `name` and `value`,
e.g.
```js
export const STATE_NAMES = {
  NO_RESPONSE: {
    name: 'No response',
    value: 7,
  },
  NA: {
    name: 'N/a or Ack',
    value: 1,
  },
  ...
};
```

This would allow references to read like `STATE_NAMES.NO_RESPONSE.value` rather than referencing an index.

Reply:
Good point.  Let's keep that in mind and maybe redo all our enums sometime.